File: Prueba y error/ejercicios de semana santa php/Thre php/Library/TerceraLey.php
<?php
namespace Library;

require_once 'LeyNewton.php';

class TerceraLey extends LeyNewton {

    public function describir(): string {
        return "Tercera Ley de Newton - Acción y Reacción";
    }

    public function ejecutar(): void {
        echo "===== " . $this->describir() . " =====<br>";
        echo "Por cada acción hay una reacción igual y opuesta.<br>";
    }
}

File: Prueba y error/ejercicios de semana santa java/three java/Library/TerceraLey.java
package Library;

//  (Acción y Reacción)
public class TerceraLey extends LeyNewton {

    public TerceraLey() {
        super("Tercera Ley de Newton - Acción y Reacción");
    }

    @Override
    public String explicarLey() {
        return "A toda acción corresponde una reacción igual y contraria.";
    }

    @Override
    public String calcular(double... valores) {
        if (valores.length < 1) {
            return "Error: se necesita una fuerza de acción para calcular la reacción.";
        }
        double fuerza = valores[0];
        return "Fuerza de acción: " + fuerza + " N\nFuerza de reacción: " + (-fuerza) + " N";
    }
}